wurst/util/Colors.wurst

```diff
@@ -202,13 +272,20 @@ public function string.toPlayerColor() returns playercolor
 			break
 	return pc
 
+function colorHSV.test()
```

Comment:
should probably be named `assert` instead of test

Reply:
Sure